### ISSUE TITLE
could not find contributing_to_black.md on github action

### ISSUE THREAD

**Issue by krux02** [psf/black#1602]
I used the github action from the readme and copy pasted into our project. The pull requests fails with the following error message [here](https://github.com/protontypes/openselery/pull/97/checks?check_run_id=985114640):

```
 Set up job2s
##[error]Could not find file '/home/runner/work/_actions/_temp_969739ad-9b09-42e5-b290-ab56cbb44a6d/_staging/psf-black-6bedb5c/docs/contributing_to_black.md'.
Current runner version: '2.272.0'
Operating System
Virtual Environment
Prepare workflow directory
Prepare all required actions
Download action repository 'actions/checkout@v2'
Download action repository 'actions/setup-python@v2'
Download action repository 'psf/black@stable'
##[error]Could not find file '/home/runner/work/_actions/_temp_969739ad-9b09-42e5-b290-ab56cbb44a6d/_staging/psf-black-6bedb5c/docs/contributing_to_black.md'.
```
**Expected behavior**

Some error message about linting or on what I did wrong to configure black itself. I cannot do anything with the error message ablove

**Environment (please complete the following information):**

Here is the PR that should introduce black: 

https://github.com/protontypes/openselery/pull/97

**Does this bug also happen on master?** I replaced stable with master in black.yml and now I get useful error messages. So no it does not happen on master.

**Comment by ichard26** [comment]
Duplicate of #1520.

This will be fixed by a new release which #1583 is all about. We moved a lot of stuff between stable and master so that's probably why the paths are broken.

**Comment by krux02** [comment]
thank you for the fast reply. Actually reporting the issue made me try out the master which in the end did solve it for me. And of course thank you for making this tool for free.
